ff2mpv.js

```diff
@@ -96,4 +98,25 @@ getOS().then(async (os) => {
   browser.browserAction.onClicked.addListener((tab) => {
     ff2mpv(tab.url);
   });
+
+  // Messages sent with browser.runtime.sendMessage (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/sendMessage) from external applications will be handle here.
+  // ref: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessageExternal
+  browser.runtime.onMessageExternal.addListener((request, sender, sendResponse) => {
+    if (!request) {
+      console.wanr('No request in external message');
+      return;
+    }
+
+    const { type, url } = request;
+    console.info('Request from:', sender);
```

Comment:
Nitpick (since we probably only need this on the user's console if they're debugging):

```suggestion
    console.debug('Request from:', sender);
```

Reply:
Yes, I agree. Added the change